### ISSUE TITLE
Dark Theme

### ISSUE THREAD

**Issue by federicoiosue** [federicoiosue/Omni-Notes#70]

**Comment by DMSalesman** [comment]
@federicoiosue Might I inquire about how this feature is coming along, seeing as four years have passed since the issue has been opened?

**Comment by federicoiosue** [comment]
@DMSalesman this issue is actually a placeholder, due to the fact that a Pro (paid) version of the app still doesn't exists. 
Im more focused on programming related tasks, hoping for good designer contributons about this topic.

**Comment by KaKi87** [comment]
You're hoping that people will give it to you for free so that users will pay you for it ? 🤔

**Comment by 2Belette** [comment]
Is this still un roadmap? Would be a nice addition to this great app

**Comment by federicoiosue** [comment]
Hi there, 

Yes it's still there! I worked mainly focusing on technical stuffs lately but the feature is still part of the roadmap.
This will have to come after the version 7.x graphic overhaul anyway.

**Comment by landry314** [comment]
Yeah, this app really needs dark mode. I can't deal with the eye strain.

**Comment by TLINDEN** [comment]
Any news on that? I'd really appreciate it :)

**Comment by landry314** [comment]
i left this a long time ago. try notesnook

**Comment by KaKi87** [comment]
Unfortunately, end-to-end encryption also means no widget...
